files/quizgrader/docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '2.1'
 
 services:
   app:
-    image: openmrs/openmrs-contrib-quizgrader:latest
+    image: openmrsinfra/openmrs-contrib-quizgrader:latest
```

Comment:
openmrsinfra/openmrs-contrib-quizgrader does not exist.
openmrsinfra/quizgrader does, but has not (yet) been pushed.